README.md

```diff
@@ -1,19 +1,102 @@
-Exchange EWS Provider is an **add-on for
-[Thunderbird](https://mozilla.org/thunderbird) and
-[Lightning](https://mozilla.org/calendar)** that provides
-**[Microsoft Exchange](http://microsoft.com/exchange) calendar
-support**.
+## Introduction
+
+Exchange Calendar is an add-on for [Thunderbird](https://mozilla.org/thunderbird)
+and [Seamonkey](https://www.seamonkey-project.org/).
+
+It extends the [Lightning](https://mozilla.org/calendar) add-on to provide
+[Microsoft Exchange](http://microsoft.com/exchange) support.
+
+Exchange Calendar extension can:
+  * Synchronize calendars, tasks, and contacts with a Microsoft Exchange server.
+  * Create, edit and update events and tasks.
+  * Display contacts from your address books and the global address list
```

Comment:
It is recommended that all bullet points that form complete sentences end with a period (full stop), so please add those here and below.